sdk/storage/azure-storage-file-datalake-stress/src/main/java/com/azure/storage/file/datalake/stress/DataLakeScenarioBase.java

```diff
@@ -9,6 +9,7 @@
 import com.azure.identity.DefaultAzureCredential;
 import com.azure.identity.DefaultAzureCredentialBuilder;
 import com.azure.perf.test.core.PerfStressTest;
+import com.azure.storage.common.ParallelTransferOptions;
```

Comment:
We should be able to remove this import now that these are not defined in the base class.